Documentation/Functions/Data.rst

```diff
@@ -46,726 +45,815 @@ Data / getText
 
-:aspect:`Example:`
-   Get content from :php:`$cObj->parameters['color']`:
+Get content from :php:`$cObj->parameters['color']`:
 
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
 
-      lib.foo.data = parameters : color
+    lib.foo.data = parameters : color
 
-.. _data-type-gettext-path:
+..  _data-type-gettext-path:
 
 path
 ----
 
-:aspect:`getText Key:`
-   path
+..  confval:: path
+    :name: data-path
 
-:aspect:`Description:`
-   Path to a file, possibly placed in an extension, returns empty if the file
-   does not exist.
+    Path to a file, possibly placed in an extension, returns empty if the file
+    does not exist.
 
-:aspect:`Example:`
-   Get path to file rsaauth.js (inside extension rsaauth) relative to siteroot:
+..  _data-type-gettext-path-example:
 
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+Example: Resolve the path to a file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-      lib.foo.data = path : EXT:rsaauth/resources/rsaauth.js
+Get path to file rsaauth.js (inside extension rsaauth) relative to siteroot:
 
-   It can also be helpful in combination with the stdWrap function
-   :ref:`stdwrap-insertData`:
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
 
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+    lib.foo.data = path : EXT:rsaauth/resources/rsaauth.js
 
-      page.headerData.10 = TEXT
-      page.headerData.10 {
-        value = <link rel="preload" href="{path : EXT:site/Resources/Public/Fonts/Roboto.woff2}" as="font" type="font/woff2" crossorigin="anonymous">
-        insertData = 1
-      }
+It can also be helpful in combination with the stdWrap function
+:ref:`stdwrap-insertData`:
 
-.. _data-type-gettext-register:
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+    page.headerData.10 = TEXT
+    page.headerData.10 {
+        data = path : EXT:site/Resources/Public/Images/logo.png
+        wrap = <a href="|">
+    }
+
+..  _data-type-gettext-register:
 
 register
 --------
 
-:aspect:`getText Key:`
-   register
+..  confval:: register
+    :name: data-register
+    :Syntax: register: [field name from the :php:`$GLOBALS['TSFE']->register`]
+
+    See :ref:`LOAD_REGISTER <cobj-load-register>`.
 
-:aspect:`Syntax`
-   register: [field name from the :php:`$GLOBALS['TSFE']->register`]
+..  _data-type-gettext-register-example:
 
-:aspect:`Description:`
-   See :ref:`LOAD_REGISTER <cobj-load-register>`.
+Example: Get the content of a register
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-:aspect:`Example:`
-   Get content from :php:`$GLOBALS['TSFE']->register['color']`:
+Get content from :php:`$GLOBALS['TSFE']->register['color']`:
```

Comment:
```suggestion
Get content from the :ref:`register <using-setting-register>`:
```
$GLOBALS['TSFE']->register is now marked as internal, so we should not mention it:
https://github.com/TYPO3/typo3/blob/main/typo3/sysext/frontend/Classes/Controller/TypoScriptFrontendController.php#L177-L180